examples/mysql/README.md

```diff
@@ -0,0 +1,88 @@
+MySQL filter example
+====================
+
+In this example, we show how the [MySQL
+filter](https://www.envoyproxy.io/docs/envoy/latest/configuration/network_filters/mysql_proxy_filter)
+can be used with the Envoy proxy. The Envoy proxy [configuration](./envoy.yaml)
+includes a MySQL filter that parses queries and collects MySQL-specific
+metrics.
+
+## Usage
+
+Build and run the containers:
+
+```console
+$ docker-compose build && docker-compose up
+```
+
+Use `mysql` to issue some commands and verify they are routed via Envoy. Note
+that the current implementation of the protocol filter is known to have
+problems with versions of the MySQL protocol post version 5.5.
```

Comment:
My bad. Can we rephrase the compatibility message to the following?

Note that the current implementation of the protocol filter was tested with MySQL v5.5. It may, however, not work with other versions of MySQL due to differences in the protocol implementation.

Reply:
Updated.